plugins/destination/plugin_testing_migrate.go

```diff
@@ -11,13 +11,14 @@ import (
 	"github.com/cloudquery/plugin-sdk/specs"
 	"github.com/google/uuid"
 	"github.com/rs/zerolog"
+	"github.com/stretchr/testify/require"
 )
 
 func tableUUIDSuffix() string {
 	return strings.ReplaceAll(uuid.NewString(), "-", "_")
 }
 
-func testMigration(ctx context.Context, p *Plugin, logger zerolog.Logger, spec specs.Destination, target *schema.Table, source *schema.Table, mode specs.MigrateMode) error {
+func testMigration(ctx context.Context, t *testing.T, p *Plugin, logger zerolog.Logger, spec specs.Destination, target *schema.Table, source *schema.Table, mode specs.MigrateMode) error {
```

Comment:
If we're getting `*testing.T` as an input param, we don't need to return `error`. Could use `assert.Contains` and a "fake" testing.T (assert gets an interface with Errorf and nothing else) or just use a separate helper function below, or just remove the returned error and fail the test instead of returning error.

Reply:
Ah, makes sense. How about I do it in a follow up PR to keep the diff on this one small?